### PR TITLE
Add missing go compiler to the e2e test

### PR DESCRIPTION
**Pull request by nunnatsa**
Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>

```release-note
None
```

### CODE REVIEW

File: .github/workflows/e2e.yaml [hunk 1]
```diff
@@ -12,6 +12,10 @@ jobs:
         uses: actions/checkout@v3
       - name: compile clusterkubevirtadm
         run: make clusterkubevirtadm-linux
+      - name: Install Go
```
Comment (rmohr) [inline .github/workflows/e2e.yaml:15]: does `make clusterkuevirtadm-linux` need a compiler? If so, the order probably has to be changed.

Comment (nunnatsa) [reply .github/workflows/e2e.yaml:15]: sure. fixing

**Pull request merged**